### ISSUE TITLE
add `nextjs` environment variable load order

### ISSUE THREAD

**Issue by mattrybin** [dotenvx/dotenvx#187]
The reason I was looking at this repository was to find an experience similar to what I've had with next.js, particularly with regards to .env files for a utility library I'm working on.

nextjs variable load order is this:
Environment variables are looked up in the following places, in order, stopping once the variable is found.
- process.env
- .env.$(NODE_ENV).local
- .env.local (Not checked when NODE_ENV is test.)
- .env.$(NODE_ENV)
- .env

https://nextjs.org/docs/app/building-your-application/configuring/environment-variables#environment-variable-load-order

It would be nice if something like this existed:
```
dotenvx run --standard -- node index.js
```

It does not have to specific like nextjs but some standard would be nice without writing a ton of stuff.

Thanks for this tool BTW, the style of the docs are great and readable :D

**Comment by motdotla** [comment]
i like this idea, but let's not call it `--standard`. 😂 (this was very much a custom nextjs pattern we recommended against)

the flag `--nextjs` would be great.

until then, you can duplicate this functionality with:

```
$ dotenvx run -f .env.development.local -f .env.local -f .env.development -f .env --- node index.js
```

that will follow the order:

* process.env
* .env.development.local
* .env.local
* .env.development
* .env

**Comment by motdotla** [comment]
pushing out shortly the `--convention` flag, that lets you load using nextjs' convention. in the future, we can support this to support other conventions.

```
$ dotenvx run --convention=nextjs -- node index.js
```

https://github.com/dotenvx/dotenvx/pull/193

**Comment by motdotla** [comment]
released as `v0.37.0`

<img width="960" alt="image" src="https://github.com/dotenvx/dotenvx/assets/3848/7163e601-1718-4067-b132-b8fcf8c5ada6">
